Readme.md

```diff
@@ -46,19 +46,19 @@ pino(transport)
 | `noverify`                     | Allow connection to server with self-signed certificates. Default: false.                                                                                                                                                     |
 | `reconnect`                    | Enable reconnecting to dropped TCP destinations. Default: false.                                                                                                                                                              |
 | `reconnectTries`               | Number of times to attempt reconnection before giving up. Default: `Infinity`.                                                                                                                                                |
-| `onSocketClose`                | The callback when the socket is closed on TCP destinations. Default: `(socketError) => socketError && process.stderr.write(socketError.message)`.                                                                             |
```

Comment:
I would prefer to not drop this feature, otherwise we will need to bump another major.

Reply:
That's my fault, I introduced it in v5.1.0 mostly to test a new feature without realizing the limitations 😞 

In practice, I don't think anyone is using it but from a semantic version point of view that's indeed a major bump. Maybe it's better to remove it as quickly as possible if we already know that we want to remove it?

Is it a big deal to release a major version? If they are other major changes that you would like to include in the next major version I would be happy to help.